tests/test_torchinductor.py

```diff
@@ -1891,6 +1891,15 @@ def fn(a, b, c):
             ],
         )
 
+    @patch.object(config.triton, "max_tiles", 2)
```

Comment:
Do we expect to restore the default `max_tiles` to 2 with this PR? I tried doing that for `pyhpc_equation_of_state` locally, but it did generate 2 kernels instead of 1. Just checking, totally fine to fix that in future.